civihr_employee_portal/views/includes/civihr_employee_portal_handler_absence_table_header_block_area.inc

```diff
@@ -0,0 +1,212 @@
+<?php
+
+/**
+ * This class contains View Handler for overriding absence_list view.
+ * It's used to generate Absence List custom table structure based
+ * on Drupal View's fields settings.
+ */
+class civihr_employee_portal_handler_absence_table_header_block_area extends views_handler_area {
+
+  /**
+   * Render the area.
```

Comment:
can you please define parameter and return value in the doc block .. or is this an auto-generated file ?